src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+html{
+  --blue: #00ACDC;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  padding: 0 50px;
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.header > a {
+  height: 40px;
+  width: 40px;
+}
+
+a:hover {
+  color: var(--blue);
+ }
+
+.is-active {
+  color: var(--blue);
+
+  &::after {
+    content: '';
+    position: absolute;
+    background-color: var(--blue);
+    width: 37px;
```

Comment:
```suggestion
    width: 100%;
```

Reply:
Hello ;]
I applied the changes you suggested. I worte a comment in the css file. I now just discovered that I can reply here ;p
anyway I wanted to say that if I'll change the width to 100% the line will take up the whole nav. If I'll change the position to relative the line get's relocated... so I don't know how to implement this change.

Reply:
Hey, ALex :)
You need to add position relative to "is-active" class and then change width to 100%

Also remove max-width from the header class